src/mots/export.py

```diff
@@ -29,6 +31,48 @@ def escape_for_rst(value: str) -> str:
     return value
 
 
+def format_paths_for_rst(
+    value: list, indent: int = DEFAULT_LIST_TABLE_INDENT, directory: Directory = None
+) -> str:
+    """Escape and format a path string (e.g. for includes or excludes)."""
+    config = directory.config_handle.config
+    try:
+        searchfox_enabled = config["export"]["searchfox_enabled"]
+    except KeyError:
+        searchfox_enabled = False
+        searchfox_base_url = None
+    else:
+        if searchfox_enabled:
+            searchfox_base_url = (
+                f"https://searchfox.org/{config['repo']}/search?q=&path="
+            )
+
+    parsed_paths = []
+    for path in value:
+        path = path.replace("*", "\\*")
+        if searchfox_base_url:
+            path = f"`{path} <{searchfox_base_url}{path}>`__"
+        parsed_paths.append(path)
+    return f"\n{' ' * indent}| " + f"\n{' ' * indent}| ".join(parsed_paths)
+
+
+def format_people_for_rst(value: list, indent: int = DEFAULT_LIST_TABLE_INDENT) -> str:
+    """Escape and format a list of people."""
+    people_base_url = "https://people.mozilla.org/s?query="
```

Comment:
Nit: this could be defined in the module instead of this function.

Reply:
Moved to settings module instead, which is probably more apt.